core/src/test/scala/org/apache/spark/deploy/ExternalShuffleServiceMetricsSuite.scala

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.deploy
+
+import scala.collection.JavaConverters._
+
+import org.apache.spark.{SecurityManager, SparkConf, SparkFunSuite}
+import org.apache.spark.util.Utils
+
+class ExternalShuffleServiceMetricsSuite extends SparkFunSuite {
+
+  var sparkConf: SparkConf = _
+  var externalShuffleService: ExternalShuffleService = _
+
+  override def beforeAll(): Unit = {
+    super.beforeAll()
+    sparkConf = new SparkConf()
+    sparkConf.set("spark.shuffle.service.enabled", "true")
```

Comment:
nit. Shall we add `import org.apache.spark.internal.config.SHUFFLE_SERVICE_ENABLED`, do the following?
```scala
- sparkConf.set("spark.shuffle.service.enabled", "true")
+ sparkConf.set(SHUFFLE_SERVICE_ENABLED, true)
```